packages/sc-toolkit/commands/init/assembly/contracts/main.ts

```diff
@@ -0,0 +1,26 @@
+// The entry file of your WebAssembly module.
+import {
+  callerHasWriteAccess,
+  generateEvent,
+  toBytes,
+} from '@massalabs/massa-as-sdk';
+import { Args } from '@massalabs/as-types';
+
+// This function is called when the contract is deployed.
+export function constructor(args: StaticArray<u8>): StaticArray<u8> {
+  // This line is important. It ensure that this function can't be called in the future.
+  // If you remove this check someone could call your constructor function and reset your SC.
+  if (!callerHasWriteAccess) {
```

Comment:
I think we should use a simpler and more meaningful alias

Reply:
It's a ABI call of massa-as-sdk `()` were missing. The naming is general to be used in other cases as it's an ABI